plugin/resolvergen/testdata/singlefile_enable_rewrite/gqlgen.yml

```diff
@@ -0,0 +1,17 @@
+schema:
+  - "testdata/schema.graphql"
+
+exec:
+  filename: testdata/singlefile_enable_rewrite/out/ignored.go
+model:
+  filename: testdata/singlefile_enable_rewrite/out/generated.go
+resolver:
+  filename: testdata/singlefile_enable_rewrite/out/resolver.go
+  type: CustomResolverType
```

Comment:
Does this work without this option?

Reply:
@clayne11 This is the unit test for rewrite features with single-file, which follows exactly the available single-file feature. As I wrote a new function, it's good to have coverage. 
On another note, without rewrite, yes, it does work in our system with Bazel. However, it does not support any functional overwriting. The developer will generate the stubs, implement however they like after the stubs. If they happened to make large schema changes and want the new corresponding resolver stubs, they have to delete resolver.go in the directory. The main difference is that single-file only focus on generating the stubs, not keeping the implementation. 
Rewrite aims to support large schema changes while retaining the implementation of single-file mode.